tests/testthat/_snaps/eval-walk.md

```diff
@@ -110,7 +110,16 @@
     Condition
       Warning:
       Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
-      Please use `any_of()` or `all_of()` instead.
+      Please use `"a"` instead of `data$a`
+
+---
+
+    Code
+      x <- select_loc(x, .data[[var]])
+    Condition
+      Warning:
+      Use of .data in tidyselect expressions was deprecated in tidyselect 1.2.0.
+      Please use `all_of("a") (or `any_of("a") instead of `.data[["a"]]`
```

Comment:
Does this substitution happen in rlang, not tidyselect? If so, I guess the best I can do is emit a generic message?

Reply:
oh yes it happens in rlang, `.data[[` is an injection operator like `!!`.